app/controllers/concerns/open_sap_connector.rb

```diff
@@ -2,5 +2,6 @@
 class OpenSAPConnector < AbstractXikoloConnector
   NAME = 'openSAP'
   ROOT_API = 'https://open.sap.com/api/'
+  ROOT_API_V2 = 'https://open.sap.com/api/v2/'
```

Comment:
What is about other Xikolo based platforms? I think, the new API should be available at most of them.